### DISCUSSION TITLE
Client Agent with no expiration time

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#2017] [Q&A] [answered]
Hi Guys!

How could I create a install agent without expiration time? If I set time with Zero, its will work?

![image](https://github.com/user-attachments/assets/654272f8-b7b6-4a5f-82f9-39fb101ef7ac)

**Comment by azulskyknight** [comment, marked as answer]
You do not, that's a huge security risk.

If you need long term agent installation support, that's a deployment.  Usually you want to use script based installation, there are options and details on these here: https://docs.tacticalrmm.com/install_agent/
